### PR TITLE
Fix several bugs in fileupload component.

### PR DESCRIPTION
**Pull request by jmesa-sistel**
###Defect Fixes
- Fix not to be able to select twice the same file even if you have press clean or remove item (clear fileinput value)
- Fix to open twice the file selector dialog in multiple mode (not apply ui-fileupload-choose-selected class style in multiple mode)

### CODE REVIEW

**Comment by jmesa-sistel** [comment]
Probably fix [#3491](https://github.com/primefaces/primeng/issues/3491) and [#3385](https://github.com/primefaces/primeng/issues/3385)

File: src/app/components/fileupload/fileupload.ts [hunk 1]
```diff
@@ -54,142 +54,148 @@ import {PrimeTemplate,SharedModule} from '../common/shared';
...
                 break;
             }
         });
     }
-        
+
+    onChooseClick(event, fileInput) {
+        fileInput.value = null;
+        fileInput.click();
+    }
```
Comment (jmesa-sistel) [inline src/app/components/fileupload/fileupload.ts]: Ignore it

**Comment by cagataycivici** [comment]
There are too many line changes, I can't see what the actual changes are, please fix the end of line problem and drop a comment and we'll review again.

**Comment by jmesa-sistel** [comment]
ok, done!

**Pull request closed without merging**